source/utilities.cc

```diff
@@ -230,7 +254,7 @@ namespace aspect
                        ExcMessage ("The required format for field <"
                                    + property_name
                                    + "> was not found. Specify a comma separated "
-                                   "list of `<double>' or `<id> : <double>'."));
+                                   "list of `<double>' or `<id> : <double>|<double|...'."));
```

Comment:
```suggestion
                                   "list of `<double>' or `<id> : <double>|<double>|...'.")); 
```
Maybe a missing `>`?